rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -61,217 +72,564 @@ class LiaisonNegotiator : public rmf_traffic::schedule::Negotiator
 } // anonymous namespace
 
 //==============================================================================
-auto FleetUpdateHandle::Implementation::estimate_delivery(
-    const rmf_task_msgs::msg::Delivery& request) const
--> rmf_utils::optional<FleetUpdateHandle::Implementation::DeliveryEstimate>
+void FleetUpdateHandle::Implementation::dock_summary_cb(
+  const DockSummary::SharedPtr msg)
```

Comment:
Nitpick: This could be a `const DockSummary::SharedPtr&` to avoid unnecessary copying and reference counting.

Reply:
Done https://github.com/osrf/rmf_core/pull/217/commits/f73f6896aedf9ba33247fe43c9034138912a848a